robot/impl/data/fake_wrongconfig4.json

```diff
@@ -11,7 +11,7 @@
         {
             "name": "cameraOver",
             "type": "camera",
-            "model": "file",
+            "model": "fake",
```

Comment:
?

Reply:
I changed this because it was causing issues in testing.  Khari said it got renamed to "image_file" so I'll rename to that